forms/aws/lambda/templates/templates.js

```diff
@@ -21,51 +31,63 @@ exports.handler = async function (event) {
     DELETE:
       - (Required) formID to delete
   */
-
+  const method = event.method;
   let formID = event.formID ? parseInt(event.formID) : null,
     formConfig = event.formConfig ? "'" + JSON.stringify(event.formConfig) + "'" : null;
 
   // if formID is NaN, assign it to an id we know will return no records
   if (isNaN(formID)) formID = 1;
```

Comment:
I'm wondering if instead of setting this to `formID = 1` we throw an Error instead and short circuit the Lambda call?
We just need to wrap the whole function body in a try-catch to gracefully handle the error and return a failure code back to the app.

Reply:
This definitely needs to be updated, since we can't guarantee we won't have a formID = 1 now that we're out of sandbox. I'm not sure it falls within the scope of *this* PR, but I will log an issue to implement this.

Reply:
https://github.com/cds-snc/platform-forms-client/issues/250